02-express-tutorial/controllers/people.js

```diff
@@ -0,0 +1,61 @@
+const {people}= require('../data')
+
+//get list of all people in array
+const getPeople = (req,res)=>{
+    res.json(people)
+}
+
+//add a new person
+const addPerson= (req,res)=>{
+    const {name} = req.body;
+    if(!name){
+        res.status(400).json({success: false, message:"Please provide a name"})
```

Comment:
Missing return so the code execution continues and an person without a name gets added.